README.md

```diff
@@ -2,31 +2,177 @@
+breakpoint. Below, a **debug console** will appear. This will allow us to
+control what happens when we set a breakpoint. In the debug console, click the
+"Java Visualizer" to show that instead of the console:
+
+![java-visualizer-debug window](https://curriculum-content.s3.amazonaws.com/java-mod-1/java-visualizer-and-memory/java-visualizer-debug-window.png)
+
+The other buttons to the right of the Java Visualizer tab that we highlighted in
+the screenshot above show how to proceed with the execution of the program. We
+will highlight each of those buttons in a later lesson. For now, click the
+step over
+(![step-over](https://curriculum-content.s3.amazonaws.com/java-mod-3/debugger/step-over.png))
+button.
+
+When we click the step over button, notice what happens in the Java Visualizer
+window:
+
+![call-stack-x](https://curriculum-content.s3.amazonaws.com/java-mod-1/java-visualizer-and-memory/call-stack-x.png)
+
+The execution of the program has moved onto line 6, but our stack now has the
+`x` variable along with the value we initialized it to, `1`. This is showing how
+the variables are being stored by Java! If we click the step over button again,
+we should see that the variable `y` and its value, `2`, is now also stored on
+the stack.
+
+![call-stack-xy](https://curriculum-content.s3.amazonaws.com/java-mod-1/java-visualizer-and-memory/call-stack-xy.png)
+
+If we click the step over button one more time, we will see all three of our
+variables, `x`, `y`, and `z` stored on the stack inside of memory.
+
+![call-stack-xyz](https://curriculum-content.s3.amazonaws.com/java-mod-1/java-visualizer-and-memory/call-stack-xyz.png)
+
+### Using the Browser Based Java Visualizer
+
+We could also copy and paste this code into the browser based Java Visualizer.
+The browser based Java Visualizer can be found here:
+
+[Java Visualizer for Variables and Call Stack Lesson](https://pythontutor.com/visualize.html#code=public%20class%20Main%20%7B%0A%20%20%20%20public%20static%20void%20main%28String%5B%5D%20args%29%20%7B%0A%20%20%20%20%20%20%20%20int%20x%20%3D%201%3B%0A%20%20%20%20%20%20%20%20int%20y%20%3D%202%3B%0A%20%20%20%20%20%20%20%20int%20z%20%3D%203%3B%0A%20%20%20%20%7D%0A%7D&cumulative=false&curInstr=1&heapPrimitives=nevernest&mode=display&origin=opt-frontend.js&py=java&rawInputLstJSON=%5B%5D&textReferences=false)
+
+For convenience, we have also embedded the Java Visualizer you would see on a
+browser here:
+
+<iframe width="800" height="500" frameborder="0" src="https://pythontutor.com/iframe-embed.html#code=public%20class%20Main%20%7B%0A%20%20%20%20public%20static%20void%20main%28String%5B%5D%20args%29%20%7B%0A%20%20%20%20%20%20%20%20int%20x%20%3D%201%3B%0A%20%20%20%20%20%20%20%20int%20y%20%3D%202%3B%0A%20%20%20%20%20%20%20%20int%20z%20%3D%203%3B%0A%20%20%20%20%7D%0A%7D&codeDivHeight=400&codeDivWidth=350&cumulative=false&curInstr=1&heapPrimitives=nevernest&origin=opt-frontend.js&py=java&rawInputLstJSON=%5B%5D&textReferences=false"> </iframe>
```

Comment:
I'm not sure this embedded properly. I see this HTML on the markdown preview

Reply:
Github won't display it correctly, but canvas will as long as IntelliJ previewer does.

Reply:
@developerjaym - see Linda's comment above. It doesn't show correctly in IntelliJ, but it does show a big blank space of where it should be embedded. Linda has tested the process in the Canvas page I shared on the se-java-team slack channel.